test/core/test_vectors/test.cpp

```diff
@@ -261,7 +262,20 @@ auto search() {
   return vectors;
 }
 
-struct TestVectors : testing::TestWithParam<MessageVector> {};
+struct TestVectors : testing::TestWithParam<MessageVector> {
+  static void SetUpTestCase() {
```

Comment:
In this case we always load proofs even if we already obtain them

Reply:
No, there is check if proofs has been already loaded in `ProofParamProvider::getParams`.